### PR TITLE
Search ux improvements

### PR DESCRIPTION
**Pull request by seanyesmunt**
Depends on https://github.com/lbryio/lbry-redux/pull/67

### Changes
Defaulting to `search` when entering on the url bar
Making the exact url stand out more on the search page.

screen shot 2018-08-15 at 12 11 18 am

### CODE REVIEW

**Comment by skhameneh** [comment]
```error An unexpected error occurred: "/home/travis/build/lbryio/lbry-desktop/package.json: Unexpected token < in JSON at position 1754".```

Looks like you have some unresolved merge conflicts checked in

**Comment by seanyesmunt** [comment]
Sorry @skhameneh this wasn't ready for a review quite yet. And I rebased on the related content branch. Forgot to remove that.

File: src/renderer/component/recommendedContent/view.jsx [hunk 1]
```diff
@@ -37,14 +37,7 @@ export default class RecommendedContent extends React.PureComponent<Props, State
...
     if (claim && claim.value && claim.value.stream && claim.value.stream.metadata) {
-      const {
-        value: {
-          stream: {
-            metadata: { title },
-          },
-        },
-      } = claim;
-
+      const { title } = claim.value.stream.metadata;
```
Comment (seanyesmunt) [inline src/renderer/component/recommendedContent/view.jsx]: Flow had an issue with the destructuring. Not sure why it's ok with this

**Comment by seanyesmunt** [comment]
@skhameneh Ready for a review for real this time 🙂

File: src/renderer/component/fileTile/view.jsx [hunk 2]
```diff
@@ -65,7 +67,7 @@ class FileTile extends React.PureComponent<Props> {
       hideNoResult,
       displayHiddenMessage,
       displayDescription,
-      small,
+      size,
```
Comment (skhameneh) [inline src/renderer/component/fileTile/view.jsx]: Nice :)

File: src/renderer/component/recommendedContent/view.jsx (hunk 1 above)
Comment (skhameneh) [reply src/renderer/component/recommendedContent/view.jsx]: This is much cleaner as well

**Pull request merged**